src/MudBlazor/Base/MudBaseButton.cs

```diff
@@ -35,7 +36,16 @@ public abstract class MudBaseButton : MudComponentBase
         /// </summary>
         [Parameter]
         [Category(CategoryTypes.Button.ClickAction)]
-        public string Link { get; set; }
+        public string Href { get; set; }
+        /// <summary>
+        /// If set to a URL, clicking the button will open the referenced document. Use Target to specify where (Obsolete replaced by Href)
+        /// </summary>
+        [Obsolete("Use Href Instead.", true)]
```

Comment:
```suggestion
        [Obsolete("Use Href Instead.", false)]
```
We want to keep this set to `false` until the next major release for backwards compability.